hearth/spec/hearth/interceptor_list_spec.rb

```diff
@@ -81,14 +90,19 @@ def read_before_execution(_ctx); end
       let(:hook) { :read_before_execution }
 
       before(:each) do
-        allow(context).to receive(:interceptor_context).and_return(ictx)
+        allow(context).to receive(:interceptor_context).and_return(i_ctx)
```

Comment:
We shouldn't need this now, right?

Reply:
Need what now?

Reply:
The rename was because of ide warning - it was annoying.

Reply:
I mean `context` should not receive `interceptor_context` anymore, since thats no longer a method on it